linux/open_vm_tools/ovt_verify_install.yml

```diff
@@ -22,13 +22,20 @@
           set_fact:
             update_vmtools: True
           when: update_vmtools is undefined
-        
+
+        # Flatcar doesn't support to install open-vm-tools
+        - include_tasks: ../../common/skip_test_case.yml
+          vars:
+            skip_msg: "Skip test case because {{ guest_os_ansible_distribution }} doesn't support installing open-vm-tools"
+            skip_reason: "Not Supported"
+          when: "'Flatcar' in guest_os_ansible_distribution"
+
         # VM has open-vm-tools installed and update_vmtools is set false
         - include_tasks: ../../common/skip_test_case.yml
           vars:
             skip_msg: "Skip test case due to condition for installing VMTools is not satisfied"
+            skip_reason: "Not Applicable"
```

Comment:
Here is the same situation as in deploy_vm test case when VM is existing and new_vm is false. I think "No Run" is better.